soc/intel/intel_adsp/tools/cavstool.py

```diff
@@ -745,6 +745,9 @@ def debug_offset():
     ( base, stride ) = adsp_mem_window_config()
     return base + stride * 2
 
+def debug_slot_offset(num):
+    return debug_offset() + DEBUG_SLOT_SIZE * (1 + num)
```

Comment:
I guess in the future we will need to detect slot size used by FW if it changes, but today using the default is good.